### ISSUE TITLE
Names for scheduler threads

### ISSUE THREAD

**Issue by tomaszglinski** [kagkarlsson/db-scheduler#23]
Since I was also working on my own scheduler, I found that post with few quite good tips.
http://www.nurkiewicz.com/2014/11/executorservice-10-tips-and-tricks.html

One of them was to name Threads so that later on undertanding the logs and debugging is much easier.

**Comment by kagkarlsson** [comment]
Thanks for the tips!

Currently the `schedulerName` is set as a prefix to the threads. What additional data would you like in the thread-names? `TaskInstance: <taskName>_<id>` might be a good idea..

**Comment by tomaszglinski** [comment]
Yes, that would be very good, but name and id should be then cleaned. Not
sure how long should be the names, usually log files truncate it to quite
short length.

2018-03-22 15:27 GMT+01:00 Gustav Karlsson <notifications@github.com>:

**Comment by kagkarlsson** [comment]
I have since renamed the scheduler threads, removing the `SchedulerName` as prefix, and hard-coded it to `db-scheduler-`. I suppose this should be added in the `ExecuteTask` class. Pull-requests are welcome :)

**Comment by tomaszglinski** [comment]
Do you need the "pool-1"? Do you use multiple task executions pools?

**Comment by kagkarlsson** [comment]
Hmm, maybe you are right, it's unnecessary. Thread-names will be unique anyway. However I have 3 threadpools, so I have a different prefix for those. 

What about: 
`db-scheduler-execute-due_<task-name>_<instance-id>`

What is max length of a thread-name? Should probably check and possibly truncate if necessary.
